src/meta/client/MetaClient.cpp

```diff
@@ -1180,8 +1182,11 @@ folly::Future<StatusOr<bool>> MetaClient::heartbeat() {
     return future;
 }
 
-folly::Future<StatusOr<int64_t>> MetaClient::balance() {
+folly::Future<StatusOr<int64_t>> MetaClient::balance(bool isStop) {
     cpp2::BalanceReq req;
+    if (isStop) {
```

Comment:
maybe no need this check, and if the isStop is false, we can set stop of request to false directly.

Reply:
That flag is a optional flag, so we only set it when it is true.